acceptance_tests/features/pages/reporting_unit.py

```diff
@@ -60,11 +64,9 @@ def click_change_response_status_link(ru_ref, survey, period):
 
 def get_unused_iac(ru_ref, survey_short_name):
     go_to(ru_ref)
+    click_data_panel(survey_short_name)
+    return browser.find_by_id(f'unused-enrolment-code-{survey_short_name}').value
```

Comment:
This will fail if there isn't an unused enrolment code instead of giving a None back like it did before, not sure if you want to handle that edge case?

Reply:
If it gives back None the test will still fail anyway? We could handle the error better though for sure, I'll have a look

Reply:
I've added `assert enrolment_code` after calls to this function